src/App.tsx

```diff
@@ -3,33 +3,57 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  query: string,
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    query: '',
+  };
+
+  handlerInput = (event: React.ChangeEvent<HTMLInputElement>) => {
+    this.setState({
+      query: event.target.value,
+    });
+  };
 
   render() {
+    const { query } = this.state;
+    const visibleMovies = moviesFromServer.filter((value) => {
+      const title = value.title.toLowerCase();
+      const description = value.description.toLowerCase();
+      const lowerCaseQuery = query.toLowerCase();
+
+      if (title.includes(lowerCaseQuery) || description.includes(lowerCaseQuery)) {
+        return true;
+      }
+
+      return false;
+    });
```

Comment:
create getVisibleMovies method
```suggestion
    const visibleMovies = this.getVisibleMovies(moviesFromServer);
```